src/samplers/aliastable.jl

```diff
@@ -15,9 +15,7 @@ end
 
 function rand(rng::AbstractRNG, s::AliasTable)
     i = rand(rng, 1:length(s.alias)) % Int
-    u = rand(rng)
-    @inbounds r = u < s.accept[i] ? i : s.alias[i]
-    r
+    ifelse(rand(rng) < s.accept[i],  i, s.alias[i])
```

Comment:
Just one final suggestion: It might be good to add a comment that explains why `ifelse` is used here. Something like
```suggestion
    # using `ifelse` improves performance here: github.com/JuliaStats/Distributions.jl/pull/1831/
    ifelse(rand(rng) < s.accept[i],  i, s.alias[i])
```

Reply:
sounds good